feature/library/src/main/java/com/ku_stacks/ku_ring/library/LibrarySeatViewModel.kt

```diff
@@ -0,0 +1,50 @@
+package com.ku_stacks.ku_ring.library
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.ku_stacks.ku_ring.library.repository.LibraryRepository
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import kotlinx.coroutines.flow.update
+import kotlinx.coroutines.launch
+import javax.inject.Inject
+
+@HiltViewModel
+class LibrarySeatViewModel @Inject constructor(
+    private val libraryRepository: LibraryRepository,
+) : ViewModel() {
+
+    private val _uiState = MutableStateFlow(LibrarySeatUiState.Empty)
+    val uiState: StateFlow<LibrarySeatUiState>
+        get() = _uiState.asStateFlow()
```

Comment:
```suggestion
    val uiState = _uiState.asStateFlow()
```

이건 get할 때마다 굳이 새로운 asStateFlow() 호출할 필요는 없을 것 같아요. 

+) asStateFlow와 ``val uiState: StateFlow<아무거나> = _uiState``
의 차이는 어떤 것이 있을까요?

Reply:
`val uiState: StateFlow<...> = _uiState`의 경우 UI 단에서 MutableStateFlow로 타입캐스팅이 가능하지 않을까 추측해봅니다.

찾아보니까 `asStateFlow`는 ReadonlyStateFlow라는 읽기 전용 플로를 제공하여 편집이 불가능한 플로를 노출시킬 수 있다고 하네요!

Reply:
이거 저한테도 알려주셨던 거 ㅋㅋ